doc-content/enterprise-only/installation/controller-jws-install-proc.adoc

```diff
@@ -41,9 +41,9 @@ ifeval::["{context}" == "install-on-tomcat"]
 endif::[]
   directory.
 +
-[NOTE] 
+[NOTE]
 ====
-Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the 
+Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the
```

Comment:
"...you copy..." Why present continuous  tense?

Reply:
Good point! Changing.